kserve/base/params.env

```diff
@@ -0,0 +1,7 @@
+kserve-controller=docker.io/skonto/kserve-manager:dev2
```

Comment:
We can revert when things are properly built with all OCP patches.

Reply:
As it is "broken" until we have the correct manifests in opendatahub-io/kserve, I omitted your changes. I hope in the mean time we can get the upstream PR in, so this should no longer be necessary.